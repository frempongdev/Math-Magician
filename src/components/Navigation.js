import React from 'react';
import { Link } from 'react-router-dom';
import style from '../styleModules/Header.module.css';
import art from '../assets/man.png';

const Navigation = () => (
  <div className={style.header}>
    <div className={style.topicBox}>
      <img className={style.man} alt="man" src={art} />
      <h1 className={style.topic}>Math Magician</h1>
    </div>
    <nav className={style.navBox}>
      <ul className={style.nav}>
        <li><Link to="/">Home</Link></li>
        <span className={style.divider}>|</span>
        <li><Link to="/Calculator">Calculator</Link></li>
        <span className={style.divider}>|</span>
        <li><Link to="/Quote">Quote</Link></li>
      </ul>
    </nav>
  </div>
);

export default Navigation;
