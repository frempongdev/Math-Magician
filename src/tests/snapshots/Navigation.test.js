import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navigation from '../../components/Navigation.js';// eslint-disable-line import/extensions

describe('Navigation', () => {
  test('should match with Navigation snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Navigation />
      </MemoryRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
