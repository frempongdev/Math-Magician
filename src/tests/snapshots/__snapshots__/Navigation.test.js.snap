// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match with Navigation snapshot 1`] = `
<div
  className="header"
>
  <div
    className="topicBox"
  >
    <img
      alt="man"
      className="man"
      src="man.png"
    />
    <h1
      className="topic"
    >
      Math Magician
    </h1>
  </div>
  <nav
    className="navBox"
  >
    <ul
      className="nav"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <span
        className="divider"
      >
        |
      </span>
      <li>
        <a
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <span
        className="divider"
      >
        |
      </span>
      <li>
        <a
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
